packages/docs/src/components/ButtonVariationsTable.tsx

```diff
@@ -0,0 +1,228 @@
+import React, { ReactElement } from 'react';
+import {
+  Badge,
+  Button,
+  Table,
+  TableCaption,
+  TableHead,
+  TableRow,
+  TableBody,
+  TableCell,
+} from '@cmsgov/design-system';
+
+interface VariationButtonUsage {
+  main?: {
+    onLight?: boolean;
+    onDark?: boolean;
+  };
+  alternate?: {
+    onLight?: boolean;
+    onDark?: boolean;
+  };
+}
+
+interface ThemeButtonUsage {
+  outline?: VariationButtonUsage;
+  solid?: VariationButtonUsage;
+  ghost?: VariationButtonUsage;
+}
+
+const themeButtonUsage: { [theme: string]: ThemeButtonUsage } = {
+  healthcare: {
+    outline: {
+      main: {
+        onDark: false,
+      },
+      alternate: {
+        onLight: false,
+        onDark: false,
+      },
+    },
+    solid: {
+      main: {
+        onDark: false,
+      },
+      alternate: {
+        onLight: false,
+        onDark: false,
+      },
+    },
+    ghost: {
+      main: {
+        onDark: false,
+      },
+      alternate: {
+        onLight: false,
+        onDark: false,
+      },
+    },
+  },
+  medicare: {
+    outline: {
+      main: {
+        onDark: false,
+      },
+      alternate: {
+        onLight: false,
+        onDark: false,
+      },
+    },
+    ghost: {
+      alternate: {
+        onLight: false,
+        onDark: false,
+      },
+    },
+  },
+};
+
+export interface ButtonVariationsTableProps {
+  /**
+   * Name of currently selected theme
+   */
+  theme: string;
+}
+
+const ButtonVariationsTable = ({ theme }: ButtonVariationsTableProps) => {
+  const uses = themeButtonUsage[theme] ?? {};
+  const unused = <Badge variation="alert">Unused</Badge>;
```

Comment:
Wondering if we can update the unused to add a badge after the heading instead of adding a line of text?
![Screen Shot 2022-07-27 at 12 14 48 PM](https://user-images.githubusercontent.com/1449852/181297442-8e7cd19c-a394-4c43-9c96-8052c0868c3b.png)

Reply:
Yeah, we could do that 👍